pytermgui/ansi_interface.py

```diff
@@ -27,6 +28,14 @@
 _IS_NT = _name == "nt"
 _SYS_HAS_FRAME = hasattr(sys, "_getframe")
 
+if _IS_NT:
+    import ctypes
+    import ctypes.wintypes
```

Comment:
I don't think this is necessary. Have you tested, and came to the conclusion it is?

Reply:
I am testing it.